content/en/docs/next/op-guide/monitoring.md

```diff
@@ -129,6 +129,23 @@ Sample dashboard:
 
 ![](../etcd-sample-grafana.png)
 
+## Distributed tracing
+
+In v3.5 etcd has added support for distributed tracing using [OpenTelemetry](https://github.com/open-telemetry).
+
+> Note: This feature is still experimental and can change at any time.
```

Comment:
We don't use notes consistently across the site, but with the new theme we do have a Note/Alert box that you can use if you're interested.

You can see it at the end of the [_Install etcd_ section](https://etcd.io/docs/next/quickstart/#install-etcd) of the _Quickstart_ page.

```go-html-template
{{% alert title="Note" color="info" %}}
To work with the latest version, learn how to [build from the main branch](/docs/{{< param version >}}/install/#build-from-source).
{{% /alert %}}
```

If you do choose to use it, please update the other notes on the page as well so it is consistent at the page level.